src/templates/cv_medicine_template.html

```diff
@@ -137,165 +138,289 @@
               <p><%= item['citation'].replace(" " + lastName," <b>" + lastName + "</b>") %></p>
           <% }); %>
          </div>
         </div>
       <% } %>
 
-      <% if (books.length > 0) { %>
+       <% if (manuscripts.length > 0) { %>
         <div class="row">
-         <div class="heading_10">Books</div>
+         <div class="heading_10">Study-related manuscriptson multicenter clinical trials</div>
          <div class="regular_10">
-          <% _.forEach(books, function(item) { %>
-             <p><%= item['citation'].replace(" " + lastName," <b>" + lastName + "</b>") %></p>
+          <% _.forEach(manuscripts, function(item) { %>
+              <p><%= item['citation'].replace(" " + lastName," <b>" + lastName + "</b>") %></p>
           <% }); %>
          </div>
         </div>
       <% } %>
 
-      <% if (bookReviews.length > 0) { %>
+       <% if (letters.length > 0) { %>
         <div class="row">
-         <div class="heading_10">Book Reviews</div>
+         <div class="heading_10">Letters</div>
          <div class="regular_10">
-          <% _.forEach(bookReviews, function(item) { %>
-            <p><%= item['citation'].replace(" " + lastName," <b>" + lastName + "</b>") %></p>
+          <% _.forEach(letters, function(item) { %>
+              <p><%= item['citation'].replace(" " + lastName," <b>" + lastName + "</b>") %></p>
           <% }); %>
          </div>
-       </div>
+        </div>
       <% } %>
 
-      <% if (bookSections.length > 0) { %>
+       <% if (editorials.length > 0) { %>
         <div class="row">
-         <div class="heading_10">Book Sections</div>
+         <div class="heading_10">Editorials</div>
          <div class="regular_10">
-          <% _.forEach(bookSections, function(item) { %>
-            <p><%= item['citation'].replace(" " + lastName," <b>" + lastName + "</b>") %></p>
+          <% _.forEach(editorials, function(item) { %>
+              <p><%= item['citation'].replace(" " + lastName," <b>" + lastName + "</b>") %></p>
           <% }); %>
          </div>
-       </div>
+        </div>
       <% } %>
 
-      <% if (bookSeries.length > 0) { %>
+       <% if (books.length > 0) { %>
         <div class="row">
-         <div class="heading_10">Book Series</div>
+         <div class="heading_10">Books</div>
          <div class="regular_10">
-          <% _.forEach(bookSeries, function(item) { %>
-             <p><%= item['citation'].replace(" " + lastName," <b>" + lastName + "</b>") %></p>
+          <% _.forEach(books, function(item) { %>
+              <p><%= item['citation'].replace(" " + lastName," <b>" + lastName + "</b>") %></p>
           <% }); %>
          </div>
-       </div>
+        </div>
       <% } %>
 
-      <% if (conferencePapers.length > 0) { %>
+       <% if (booksections.length > 0) { %>
         <div class="row">
-         <div class="heading_10">Conference Papers</div>
+         <div class="heading_10">Book sections</div>
```

Comment:
for this and other headings I suspect they'll want all words capitalized, ex: "Book Sections"